CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ AppEngine version, listed here to ease deployment and troubleshooting.
 
 ## Next Release (replace with git tag when deployed)
 
+ * Made search accessible via an OpenSearch description file at `/osd.xml`.
```

Comment:
In general we put items here that we need to look out when we are deploying a new version. This is not a risk factor, it can be removed.